webapi/src/test/scala/org/knora/webapi/responders/admin/ProjectsServiceLiveSpec.scala

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright © 2021 - 2022 Swiss National Data and Service Center for the Humanities and/or DaSCH Service Platform contributors.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.knora.webapi.responders.admin
+
+import zio._
+import zio.mock._
+import zio.prelude.Validation
+import zio.test._
+
+import dsp.valueobjects.Iri._
+import dsp.valueobjects.Project.ShortCode
+import dsp.valueobjects.Project._
+import dsp.valueobjects.V2
+import org.knora.webapi.messages.admin.responder.projectsmessages.ProjectADM
+import org.knora.webapi.messages.admin.responder.projectsmessages.ProjectCreatePayloadADM
+import org.knora.webapi.messages.admin.responder.projectsmessages.ProjectCreateRequestADM
+import org.knora.webapi.messages.admin.responder.projectsmessages.ProjectGetRequestADM
+import org.knora.webapi.messages.admin.responder.projectsmessages.ProjectGetResponseADM
+import org.knora.webapi.messages.admin.responder.projectsmessages.ProjectIdentifierADM
+import org.knora.webapi.messages.admin.responder.projectsmessages.ProjectOperationResponseADM
+import org.knora.webapi.messages.admin.responder.projectsmessages.ProjectsGetRequestADM
+import org.knora.webapi.messages.admin.responder.projectsmessages.ProjectsGetResponseADM
+import org.knora.webapi.messages.store.triplestoremessages.StringLiteralV2
+import org.knora.webapi.messages.util.KnoraSystemInstances
+import org.knora.webapi.responders.ActorToZioBridge
+import org.knora.webapi.responders.ActorToZioBridgeMock
+import org.knora.webapi.responders.admin.ProjectsService
+
+object ProjectsServiceLiveSpec extends ZIOSpecDefault {
+
+  private val expectNoInteraction = ActorToZioBridgeMock.empty
+
+  val layers = ZLayer.makeSome[ActorToZioBridge, ProjectsService](ProjectsService.live)
+
+  /**
+   * Creates a [[ProjectADM]] with empty content or optionally with a given ID.
+   */
+  private def getProjectADM(id: String = "") =
```

Comment:
The parameter is never used, this could be a `val`, which would make reading the references further down much clearer.

Reply:
right... it was used at some point, but then things changed